plugins/chat/config/routes.rb

```diff
@@ -7,7 +7,7 @@
     get "/channels/me" => "current_user_channels#index"
     post "/channels" => "channels#create"
     delete "/channels/:channel_id" => "channels#destroy"
-    put "/channels/:channel_id" => "channels#update"
+    put "/channels/:channel_id" => "channels#update", :constraints => { channel_id: /\d+/ }
```

Comment:
This is required because otherwise it conflicts with the `channels/read` route

Reply:
I think you just have to put read route first no?

Reply:
Yes that seems to work, thanks :+1: